packages/backend-function/src/resolve_secret_banner.ts

```diff
@@ -1,15 +1,15 @@
 /* The code in this line replaces placeholder text in environment variables for secrets with values fetched from SSM, this is a noop if there are no secrets */
 import { SSM } from '@aws-sdk/client-ssm';
-import type { AmplifySecretPaths } from './function_env_translator.js';
+import type { SsmEnvVars } from './function_env_translator.js';
```

Comment:
if this goes to customer's function then this type should have long non ambiguous name to minimize collisions.
`AmplifyFunctionSecretEnvironmentVariables` or something more unique.

Reply:
It's just a type so it's erased from the transpiled output that gets put in the lambda bundle.